cardano-cli/golden/files/golden/help/governance_action_create-no-confidence-motion.cli

```diff
@@ -0,0 +1,15 @@
+Usage: cardano-cli governance action create-no-confidence-motion --epoch EPOCH
+            --drep-cold-verification-key-file FILE
```

Comment:
any ada holder can create governance actions. We need to change this to use `stake.vkey `

Reply:
Using `StakeVerifier` instead.